sql/2019/09_Accessibility/09_07.sql

```diff
@@ -0,0 +1,2 @@
+09_07
+see 04_19 
```

Comment:
Would like to see a 09_07 specific query here

Reply:
Per earlier comment - keeping these together as the queries are very similar.